charts/mediawiki/templates/_helpers.tpl

```diff
@@ -66,10 +66,32 @@ Common deployment environment variables
       name: {{ .Values.mw.redis.passwordSecretName | quote }}
       key: {{ .Values.mw.redis.passwordSecretKey | quote }}
 {{- end }}
-- name: MW_ELASTICSEARCH_HOST
-  value: {{ .Values.mw.elasticsearch.host }}
-- name: MW_ELASTICSEARCH_PORT
-  value: {{ .Values.mw.elasticsearch.port | quote }}
+- name: MW_PRIMARY_ELASTICSEARCH_HOST
```

Comment:
I was thinking this could be less intrusive if we'd keep the `MW_ELASTICSEARCH_HOST` for the primary and then _add_ something optional like `MW_ELASTICSEARCH_SECONDARY_HOST`. That way we could roll out changes more gradually, e.g. merge this chart change without having to sync it with anything else.

Reply:
> Similar to what I had in mind but still moving from MW_ELASTICSEARCH_HOST to MW_<new something>_ELASTICSEARCH_HOST via deprecation.

Mind elaborating what you mean by "via deprecation"?

If we want to be flexible on the versions, we can also have something like `MW_ELASTICSEARCH_VERSION` and `MW_ELASTICSEARCH_SECONDARY_VERSION` I guess?

Reply:
I was meaning that the image would work using both `MW_ELASTICSEARCH_HOST` and `MW_ELASTICSEARCH_<new>_HOST` however we'd mark `MW_ELASTICSEARCH_HOST` as deprecated.

However the functionality of these two host configurations slightly differ. The old one remains functioning like it does now: it always adds the ES6 compatibility layer. The new one defaults to no compatibility layer unless it is specified.

Reply:
I pushed a PR to the mediawiki repository that may make these comments more concrete: https://github.com/wbstack/mediawiki/pull/362